homework-submission/week1/app.js

```diff
@@ -0,0 +1,110 @@
+const Header = props => {
+  return <h3 className="text-center text-danger mt-4">{props.title}</h3>;
+};
+
+const StaticList = ({ done, description, deadline }) => {
+  return (
+    <li className="bg-light list-group-item">
+      {description}
+      {'  ,'}
+      {deadline}
+    </li>
+  );
+};
+
+const DynamicList = props => {
+  return (
+    <li
+      className={
+        props.done
+          ? ' bg-light list-group-item d-flex justify-content-between done'
+          : 'bg-light list-group-item d-flex justify-content-between'
+      }
+    >
+      {props.description}
+      {'  ,'}
+      {props.deadline}
+      {props.done}
+      <span>
+        <input type="checkbox" onChange={props.handleComplete} />
+      </span>
+    </li>
+  );
+};
+
+class App extends React.Component {
+  constructor(props) {
```

Comment:
props is also never used , here is a nice article about the difference between super() and super(props) 
https://overreacted.io/why-do-we-write-super-props/